apis/core/lib/handlers/csv-source.ts

```diff
@@ -37,10 +38,10 @@ export const post = labyrinth.protectedResource(
   }),
 )
 
-export const getCSVSource = labyrinth.protectedResource(
+const getCSVSource = labyrinth.protectedResource(
   shaclValidate,
   asyncMiddleware(async (req, res, next) => {
-    if (req.headers['content-type'] !== 'text/csv') {
+    if ((!req.headers['content-type'] || !is(req.headers['content-type'], 'text/csv'))) {
```

Comment:
This would be the simplest approach
```suggestion
    if (!typeis(req, 'text/csv')) {
```